### DISCUSSION TITLE
Welcome to OpenSource-uTube Discussions!

### DISCUSSION THREAD

**Discussion by TallerThanShort** [Common-Codes/uTube#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by jodri-code** [comment]
this'll be fun
